nuxeo-routing-rest-api/src/test/java/org/nuxeo/ecm/restapi/server/WorkflowEndpointTest.java

```diff
@@ -1086,6 +1110,19 @@ public void testTaskWorkflowInfo() throws IOException {
             assertEquals(1, node.get("entries").size());
             JsonNode taskNode = node.get("entries").elements().next();
             assertEquals(createdWorkflowInstanceId, taskNode.get("workflowInstanceId").textValue());
+            assertFalse(taskNode.get("allowTaskReassignment").booleanValue());
```

Comment:
Would there be a way of testing a case where `allowTaskReassignment` is true?

Reply:
Yes, i think so.
It is part of the improvements I'm trying to do. Using the default workflows, we can advance to a specific task that allows reassignment (e.g. "Validate The Document")